provider/resources.go

```diff
@@ -136,7 +137,12 @@ func Provider() tfbridge.ProviderInfo {
 			},
 		},
 		DataSources: map[string]*tfbridge.DataSourceInfo{
-			"http": {Tok: "http:index/getHttp:getHttp"},
+			"http": {
+				Tok: "http:index/getHttp:getHttp",
+				Docs: &tfbridge.DocInfo{
+					Markdown: readFile("upstream/docs/data-sources/http.md"),
```

Comment:
This is a scary workaround, since this is called regardless of if we are part of tfgen or tfbridge.

Reply:
This is what the tests are for apparently. This failed the acceptance test.